README.md

```diff
@@ -1,54 +1,54 @@
-# Personal-Portofolio
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Portofolio setup and mobile version skeleton
-
-> A personal portofolio project to learn how to use basically HTML and CSS.
-
-
-## Built With
-
-- HTML
-- CSS
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Deployment
-
-
-
-## Author
-
-👤 **Author1**
-
-- GitHub: [@https://github.com/mugishayannick](https://github.handle)
-- Twitter: [@https://twitter.com/mugishayann23](https://twitter.com/mugishayann23)
-- LinkedIn: [https://www.linkedin.com/in/mugisha-yannick-1265361a1/](https://www.linkedin.com/in/mugisha-yannick-1265361a1/)
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
-
-Give a ⭐️ if you like this project!
-
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
+# Personal-Portofolio
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portofolio setup and mobile version skeleton
+
+> A personal portofolio project to learn how to use basically HTML and CSS.
+
+
+## Built With
+
+- HTML
+- CSS
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Deployment
+
+
+
+## Author
+
+👤 **Author1**
+
+- GitHub: [@https://github.com/mugishayannick](https://github.handle)
+- Twitter: [@https://twitter.com/mugishayann23](https://twitter.com/mugishayann23)
+- LinkedIn: [https://www.linkedin.com/in/mugisha-yannick-1265361a1/](https://www.linkedin.com/in/mugisha-yannick-1265361a1/)
+

```

Comment:
[OPTIONAL] I suggest you ensure that you use the new [Readme Template](https://github.com/microverseinc/readme-template/blob/master/README.md)
I have only approved this project, since I see it was already approved earlier, otherwise it is invalid. Kindly ensure that for subsequent projects, you use the new template.